testData/testConfigForViper.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Suggest to move this file to config package.

Reply:
moved